src/common/WalletBalances.vue

```diff
@@ -80,15 +79,19 @@ const web3ProvidersStore = useWeb3ProvidersStore()
 const balances = computed<Balance[]>(() => [
   {
     logoIconName: ICON_NAMES.steth,
-    value: web3ProvidersStore.balances.stEth
-      ? `${formatEther(web3ProvidersStore.balances.stEth)} stETH`
+    value: web3ProvidersStore.balances.depositToken?._isBigNumber
+      ? `${formatEther(web3ProvidersStore.balances.depositToken)} ${
+          web3ProvidersStore.depositTokenSymbol
+        }`
       : '',
```

Comment:
Does it require explicit checking `isBigNumber`?

If `depositToken` can be undefined you can just check 

```ts
value: web3ProvidersStore.balances.depositToken ? `${formatEther(web3ProvidersStore.balances.depositToken)} ${
          web3ProvidersStore.depositTokenSymbol
}` : '',
```

Reply:
done